redact-wasm/edge-light.ts

```diff
@@ -0,0 +1,55 @@
+import * as core from "./wasm/arcjet_analyze_bindings_redact.component.js";
+import type {
+  ImportObject,
+  RedactedSensitiveInfoEntity,
+  RedactSensitiveInfoConfig,
+  SensitiveInfoEntity,
+} from "./wasm/arcjet_analyze_bindings_redact.component.js";
+
+import componentCoreWasm from "./wasm/arcjet_analyze_bindings_redact.component.core.wasm?module";
+import componentCore2Wasm from "./wasm/arcjet_analyze_bindings_redact.component.core2.wasm?module";
+import componentCore3Wasm from "./wasm/arcjet_analyze_bindings_redact.component.core3.wasm?module";
+import type { ArcjetRedactCustomRedact } from "./wasm/interfaces/arcjet-redact-custom-redact.js";
```

Comment:
Nit: move this near the other type/core imports and not near the wasm module imports